.vscode/settings.json

```diff
@@ -30,10 +30,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[css]": {
-    "editor.defaultFormatter": "biomejs.biome",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.biome": "explicit"
-    }
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
wait, don’t we use Biome for CSS now? I think we want to use Biome where possible.